ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/RD-RT-ADMIN-SUBFIELD-L3LEAF8.yml

```
@@ -0,0 +1,6 @@
+---
+
+overlay_rd_type:
+# Invalid ipv4 address in admin_subfield, it will return router_id
```

so I understand this is how it behaves today but this feels wrong to me that we would silently ignore a potential input typo and replace it with router_id. Feels like this would be quite hard to tshoot.

---

Please add an issue so we can improve for 6.0.

---

Created issue - https://github.com/aristanetworks/avd/issues/5510